frontend/appflowy_flutter/lib/user/presentation/screens/sign_in_screen/widgets/sign_in_anonymous_button.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:appflowy/core/config/kv.dart';
+import 'package:appflowy/core/config/kv_keys.dart';
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/startup/entry_point.dart';
+import 'package:appflowy/startup/startup.dart';
+import 'package:appflowy/user/application/historical_user_bloc.dart';
+import 'package:appflowy/user/application/sign_in_bloc.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flowy_infra/size.dart';
+import 'package:flowy_infra_ui/widget/rounded_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class SignInAnonymousButton extends StatelessWidget {
+  final bool isMobile;
+
+  /// Used in DesktopSignInScreen and MobileSignInScreen
+  const SignInAnonymousButton({
```

Comment:
I edited the same button here. You might need to rebase after this PR get merged: https://github.com/AppFlowy-IO/AppFlowy/pull/3250